static/js/tools/map/context.ts

```diff
@@ -28,6 +28,12 @@ interface Setter<T> {
   (value: T): void;
 }
 
+export interface ParentPlace {
```

Comment:
please add comments for shared interfaces

Reply:
done